docs/api/api_blueprint.apib

```diff
@@ -6530,3 +6530,25 @@ A temporary cropped image is first uploaded using this endpoint. After saving th
         {
             "url":"/serve_static/media/temp/images/6a6614c7-d458-43de-a9bd-5c329e273bf0/U0lLbUt2SF/4d4dae9e-0bab-4e72-ba2f-e8e633a145e5.gif"
         }
+
+## File Upload [/v1/upload/file{?force_local}]
+
+This endpoint supports uploading of any kind of file or multiple files. If a single file is to be uploaded, send a single file in the keyword **file**, else if multiple files, send an array of files in the keyword **files**. After saving in it's prefered storage (local, google storage, s3, etc.), the URL or array of URLs returned as response. *`force_local` parameter can be set to __true__ to force storage in local.*
+
+
+### Upload a File [POST]
+
++ Parameters
+    + force_local (optional, string, `true`) - Force storage of temporary image in local
+
++ Request(multipart/form-data; boundary=---BOUNDARY)
+
+        -----BOUNDARY
```

Comment:
@niranjan94 as far as I saw in examples of API Blueprint of files, if the data is in the form of base64encoded image or any other hash, then data is also included. Else, it is shown like this. One of the links I referred is [this](https://github.com/apiaryio/dredd/issues/129#issue-54313115). See the CSV file upload in the example.